src/utils/env.ts

```diff
@@ -71,6 +71,16 @@ export const env = createEnv({
     RETRY_TX_CRON_SCHEDULE: z.string().default("*/30 * * * * *"),
     MAX_BLOCKS_ELAPSED_BEFORE_RETRY: z.coerce.number().default(10),
     MAX_WAIT_TIME_BEFORE_RETRY: z.coerce.number().default(600),
+    WEBHOOKS_ENABLED: boolSchema("true"),
```

Comment:
What would be the reason to make this disable-able

Reply:
i already do a URL check so this is redundant, will remove. thanks.